src/CMakeLists.txt

```diff
@@ -2,6 +2,22 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+set(CRC32C_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+set(CRC32C_BUILD_BENCHMARKS OFF CACHE BOOL "" FORCE)
+set(CRC32C_USE_GLOG OFF CACHE BOOL "" FORCE)
+set(CRC32C_INSTALL OFF CACHE BOOL "" FORCE)
+# Honor visibility properties for all target types.
+# Required for Crc32c sub-project as it has the CMake minimum
+# required version 3.1.
```

Comment:
How much cleaner is this going to be if we just bump the CMake minimum required version in the [crc32c project](https://github.com/bitcoin-core/crc32c-subtree)?

Reply:
> How much cleaner is this going to be if we just bump the CMake minimum required version in the [crc32c project](https://github.com/bitcoin-core/crc32c-subtree)?

Just one LOC `set(CMAKE_POLICY_DEFAULT_CMP0063 NEW)` will gone.

Much cleaner it would be if defaults of `CRC32C_*` options are set to `OFF` in the [crc32c project](https://github.com/bitcoin-core/crc32c-subtree).

Reply:
This code has been [dropped](https://github.com/hebasto/bitcoin/pull/6#issuecomment-1436835797).